boards/arm/adafruit_grand_central_m4_express/doc/index.rst

```diff
@@ -0,0 +1,194 @@
+| Interface | Controller | Driver/Component                         |
++===========+============+==========================================+
+| NVIC      | on-chip    | Nested vector interrupt controller       |
++-----------+------------+------------------------------------------+
+| SYSTICK   | on-chip    | SysTick                                  |
++-----------+------------+------------------------------------------+
+| WDT       | on-chip    | Watchdog                                 |
++-----------+------------+------------------------------------------+
+| GPIO      | on-chip    | I/O ports, User LED                      |
++-----------+------------+------------------------------------------+
+| UART      | on-chip    | Serial ports, Console                    |
++-----------+------------+------------------------------------------+
+| SPI       | on-chip    | SPI ports, microSDHC slot                |
++-----------+------------+------------------------------------------+
+| TRNG      | on-chip    | True Random Number Generator             |
++-----------+------------+------------------------------------------+
+| RTC       | on-chip    | Real-Time Counter                        |
++-----------+------------+------------------------------------------+
+| USB       | on-chip    | USB device                               |
++-----------+------------+------------------------------------------+
+| WDT       | on-chip    | Watchdog Timer                           |
++-----------+------------+------------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig file
+:zephyr_file:`boards/arm/adafruit_grand_central_m4_express/adafruit_grand_central_m4_express_defconfig`.
+
+Connections and IOs
+===================
+
+The `Adafruit Learning System`_ has detailed information about
+the board including `pinouts`_ and the `schematics`_.
+
+System Clock
+============
+
+The SAMD51 MCU is configured to use the 32.768 kHz external oscillator
+with the on-chip PLL generating the 120 MHz system clock.
+
+Serial Port
+===========
+
+The SAMD51 MCU has 8 SERCOM based UARTs. On the Grand Central, SERCOM0 is
+the Zephyr console and is available on RX(PB25) and TX(PB24).
+
+SPI Port
+========
+
+The SAMD51 MCU has 8 SERCOM based SPIs. On the Grand Central, SERCOM7 has been
+set into SPI mode to connect to devices over the SCK(PD09), MOSI(PD08), and MISO(PD11) pins.
+Additionally SERCOM2 has been configured as SPI to access the microSDHC card.
+
+I2C Port
+========
+
+The SAMD51 MCU has 8 SERCOM based I2Cs. On the Grand Central, SERCOM3 has been
+configured as I2C to connect to devices over the SCL(PB21) and SDA(PB20) pins.
+
+USB Device Port
+===============
+
+The SAMD51 MCU has a USB device port that can be used to communicate
+with a host PC.  See the :ref:`usb-samples` sample applications for
+more, such as the :ref:`usb_cdc-acm` sample which sets up a virtual
```

Comment:
```suggestion
more, such as the :zephyr:code-sample:`usb-cdc-acm` sample which sets up a virtual
```

Reply:
When I took this from the other examples, this role wasn't used there.
But if this is the right way I'll change that.

Once there is a review of the device tree, I will add all this small stuff as well.